### DISCUSSION TITLE
我的Famland 界面优化

### DISCUSSION THREAD

**Discussion by otto-001** [famland/demo#16] [Ideas]
[](https://user-images.githubusercontent.com/97232831/161210936-7337c5da-50f0-44b5-923e-3c17e845d0f0.png)在我的Famland 界面搞个统计，
显示合计多少田，多少收益。
搞个排序，按肥力值。共享值，加成系数筛选排序
![image](https://user-images.githubusercontent.com/97232831/161211697-a6e93027-1c87-478c-8c86-f75a246c0fa9.png)

**Comment by yanzhiwuli** [comment]
还keyi 加上，这个账户下面的FAMLAND的总价值。让玩家一目了然，知道自己有多少钱
